travis-ci/functions.sh

```diff
@@ -69,4 +69,32 @@ _install_gcc_and_ccache()
 
   info "Installing 'gcc'"
   export "PATH=${PATH}:${gcc_path}/bin"
+
+  arm-none-eabi-gcc --version
+  pip install --upgrade cmake
+}
+
+_config_and_build()
+{
+    echo “” > mbed-os.lib
+
+    mbedtools checkout
```

Comment:
Should the following be part of the cloning dependencies function instead?

```
echo “” > mbed-os.lib
mbedtools checkout
```

Reply:
Fixed